cv-data-service-library/src/test/java/com/trihydro/library/helpers/CdotGisConnectorTest.java

```diff
@@ -0,0 +1,59 @@
+package com.trihydro.library.helpers;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+
+import com.trihydro.library.service.BaseServiceTest;
+
+class CdotGisConnectorTest extends BaseServiceTest {
+
+  @InjectMocks
+  private CdotGisConnector uut;
+
+  private final String expectedBaseUrl = "https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded";
+
+  @Test
+  void testInitialization() {
+    Assertions.assertNotNull(uut);
+    Assertions.assertEquals(expectedBaseUrl, uut.getBaseUrl());
+    Assertions.assertNotNull(uut.getRestTemplateProvider());
+  }
+
+  @Test
+  void testGetBaseUrl() {
+    Assertions.assertEquals("https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes_withDEC/MapServer/exts/CdotLrsAccessRounded", uut.getBaseUrl());
```

Comment:
Should we just use the `expectedBaseUrl` value from above?

Reply:
That would make sense! Modified this to use `expectedBaseUrl` instead of a hardcoded string.